src/main/java/com/team701/buddymatcher/domain/users/User.java

```diff
@@ -0,0 +1,59 @@
+package com.team701.buddymatcher.domain.users;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Entity
+@Table(name = "USER")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private UUID id;
+
+    @Column(name = "USER_NAME", nullable = false)
+    private String name;
+
+    @Column(name = "USER_EMAIL", nullable = false)
+    private String email;
+
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    @JoinColumn(name = "PAIR_ID")
+    private Pair pair;
+
+    public UUID getId() {
+        return id;
+    }
+
+    public User setId(UUID id) {
+        this.id = id;
+        return this;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public User setName(String name) {
+        this.name = name;
+        return this;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public User setEmail(String email) {
+        this.email = email;
+        return this;
+    }
+
+    public Pair getPair() {
+        return pair;
+    }
+
+    public User setPair(Pair pair) {
```

Comment:
Is it worth maybe checking that the pair being passed in contains this user? Not sure if that type of error could ever come up though so idk

Reply:
I think this check would be useful for any future endpoints or services we create that set or update the user's pair, but I personally prefer to keep the logic for Entity setters/getters as simple as possible.

Reply:
That makes sense, sounds good